source/jquery.flot.legend.js

```diff
@@ -210,13 +216,11 @@
                     '/>';
                 break;
             default:
-                // default is circle
-                html = '<use xlink:href="#circle" class="legendIcon" ' +
+                // default is box
+                html = '<use xlink:href="#box" class="legendIcon" ' +
```

Comment:
Why are you changing the default? The default symbol for points [is 'circle'](https://github.com/flot/flot/blob/e8722748a774ab0e0229046e6121a6d8123536e4/source/jquery.flot.js#L156).

Edit: I think I understand why you're doing this, but see above comment for an alternative approach, which would allow us to keep this default intact.